### PR TITLE
add metric op

### PR DESCRIPTION
**Pull request by astorfi**
This pull request is aimed to create a metric_op for TF-Slim learning to be able to show the streaming metrics(accuracy,...) for the training phase.
The op reset the total and count local variables defined by streaming_metric(if it is streaming_accuracy ,...) per each epoch of training so the accumulative accuracy per epoch can be represented.

### CODE REVIEW

**Comment by sguada** [comment]
Please revert all the indentation changes, use 2 spaces per indentation level.

File: tensorflow/contrib/slim/python/slim/learning.py [hunk 1]
```diff
@@ -250,6 +250,8 @@ def InitAssignFn(sess):
 import os
 import sys
 import time
+import tensorflow as tf
```
Comment (sguada) [inline tensorflow/contrib/slim/python/slim/learning.py]: Don't import tensorflow directly only what is needed.

**Review by sguada — CHANGES_REQUESTED** [review]
Revert all the indentation changes, currently it is hard to see what you actually change.
Use 2 spaces indentation.

**Comment by drpngx** [comment]
@astorfi could you rebase and push again?

**Comment by astorfi** [comment]
@drpngx There were some issues regarding my pool request. I am trying to solve them yet.

**Comment by vrv** [comment]
Ping for @astorfi ...

**Pull request closed without merging**